static/js/compose.js

```diff
@@ -647,7 +670,7 @@ exports.handle_keydown = function (event) {
             }
         }
         if (isLink) {
-            // ctrl + l: Insert a link to selected text
+            // ctrl + l + shift: Insert a link to selected text
```

Comment:
This should be changed when we changed the shortcut, it was my mistake. Won't happen again.